ds-dash/files/docker/Dockerfile

```diff
@@ -0,0 +1,73 @@
+FROM python:3.7.0-slim
```

Comment:
Why do we not go over centos or redhat? I thought this the way to go?

Reply:
We had a conversation in the past in another thread and users prefer using official images rather than self-backed, which are indeed harder to maintain. Anyway, one can always switch the defaults to any custom requirement.